tools/build_project_book.py

```
@@ -0,0 +1,313 @@
+    out = []
+    if show_provenance:
+        out.append(f'<div class="doc-provenance"><strong>Operational Reference Guide:</strong> <code>{rel_path}</code></div>\n')
+
+    if "description" in metadata and metadata["description"]:
+        desc = metadata["description"]
+        out.append(f'<div class="callout callout-note">\n<strong>📌 Executive Summary</strong>\n\n{desc}\n</div>\n')
+
+    in_code_block = False
+    for line in clean.splitlines():
+        stripped = line.strip()
+        if stripped.startswith("```") or stripped.startswith("~~~"):
```

Ruff (0.16.3)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)